driver/razerkbd_driver.c

```diff
@@ -285,13 +327,42 @@ static void razer_set_device_mode(struct usb_device *usb_dev, unsigned char mode
         report.transaction_id.id = 0x3F;
         break;
     case USB_DEVICE_ID_RAZER_BLACKWIDOW_ELITE:
+    case USB_DEVICE_ID_RAZER_BLACKWIDOW_V3_MINI:
         report.transaction_id.id = 0x1F;
         break;
+    case USB_DEVICE_ID_RAZER_BLACKWIDOW_V3_MINI_WIRELESS:
+        report.transaction_id.id = 0x9F;
+        break;
```

Comment:
Interesting that they use different transaction ids... Well if it works it's okay and 0x9f also makes sense :D

Reply:
Done